tests/kafkatest/version.py

```diff
@@ -61,7 +60,7 @@ def get_version(node=None):
         return DEV_BRANCH
 
 DEV_BRANCH = KafkaVersion("dev")
-DEV_VERSION = KafkaVersion("1.1.1-SNAPSHOT")
+DEV_VERSION = KafkaVersion("1.1.2-SNAPSHOT")
```

Comment:
\cc @lindong28 -- seem's you forgot to update this when dumping the version in `1.1` branch.

Reply:
@mjsax Ah my bad. Thanks for letting me know. I will take care next time.

I just looked through the release process again and realized that I also forgot to update the version in kafka-merge-pr.py. Can you review https://github.com/apache/kafka/pull/5413?

I also slightly updated release process in https://cwiki.apache.org/confluence/pages/diffpagesbyversion.action?pageId=34840886&selectedPageVersions=139&selectedPageVersions=140.